src/results.jl

```diff
@@ -106,6 +117,24 @@ mutable struct Result
     end
 end
 
+function Result(
+    result::Ptr{libpq_c.PGresult},
+    jl_conn::Connection;
+    column_types::Union{AbstractDict,AbstractVector}=ColumnTypeMap(),
+    type_map::AbstractDict=PQTypeMap(),
+    conversions::AbstractDict=PQConversions(),
+    not_null=false,
+)
+    return Result{TEXT}(
+        result,
+        jl_conn;
+        column_types=column_types,
+        type_map=type_map,
+        conversions=conversions,
+        not_null=not_null,
+    )
+end
+
```

Comment:
```suggestion
Result(args...; kwargs...) = Result{TEXT}(args...; kwargs)

```

This way we don't have to update this method of the arguments of the constructor change.